modules/javafx.media/src/main/native/gstreamer/plugins/mfwrapper/mfwrapper.h

```diff
@@ -94,6 +94,7 @@ struct _GstMFWrapper
 
     BYTE *header;
     gsize header_size;
+    gboolean send_header;
```

Comment:
may be rename as `is_send_header` similar to the other boolean variables of the class.